### PR TITLE
copy address and link to ic rocks

### PR DESCRIPTION
**Pull request by ChriLnth**
## Changelog

- Link to ic.rocks on activity
- Show full account ID / credentials on activity feed

### Type of changes included:

- [ ] Components
- [ ] Business Logic
- [ ] Bug fixes
- [x] Styling
- [ ] Code Refactor

### Clubhouse Tickets Related:

- [CH23258](https://app.clubhouse.io/terminalsystems/story/23258/enhancement-show-full-account-id-credentials-on-activity-feed)
- [CH23258](https://app.clubhouse.io/terminalsystems/story/23258/enhancement-show-full-account-id-credentials-on-activity-feed)

### Notes:

*Place special notes here*

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: source/ui/ActivityItem/index.jsx [hunk 1]
```diff
@@ -39,13 +43,23 @@ const getStatus = (status, classes, t) => {
...
-  if (type === ACTIVITY_TYPES.RECEIVE) subtitle += ` · ${t('activity.subtitle.from')}: ${shortAddress(from)}`;
-
-  return subtitle;
+const getDate = (status, date) => (
+  status === ACTIVITY_STATUS.COMPLETED
+    ? moment(date).format('MMMM Do')
+    : ''
+);
+
+const getSubtitle = (type, to, from, t) => (
```
Comment (rocky-fleek) [inline source/ui/ActivityItem/index.jsx:52]: why not do an object to map type -> text?

File: source/ui/ActivityItem/index.jsx [hunk 2]
```diff
@@ -61,14 +75,48 @@ const ActivityItem = ({
...
+
+  const [showTooltip, setShowTooltip] = useState(false);
+  const [tooltipText, setTooltipText] = useState(copyText);
+
+  const handleClickCopy = (e) => {
+    e.stopPropagation();
+
+    /* eslint-disable no-nested-ternary */
+    navigator.clipboard.writeText(
+      type === ACTIVITY_TYPES.SEND
```
Comment (rocky-fleek) [inline source/ui/ActivityItem/index.jsx:101]: same

File: source/ui/ActivityItem/index.jsx [hunk 3]
```diff
@@ -104,23 +158,50 @@ const ActivityItem = ({
...
       <div className={classes.rightContainer}>
-        <Typography variant="h5">
-          <NumberFormat value={showSwap ? swapData.amount : amount} displayType="text" thousandSeparator="," suffix={` ${showSwap ? swapData.currency.value : currency.value}`} decimalScale={5} />
-        </Typography>
-        <Typography variant="subtitle2">
-          <NumberFormat value={showSwap ? swapData.value : value} displayType="text" thousandSeparator="," prefix="$" suffix=" USD" decimalScale={2} />
-        </Typography>
+        <div>
+          <Typography variant="h5">
+            <NumberFormat value={showSwap ? swapData.amount : amount} displayType="text" thousandSeparator="," suffix={` ${showSwap ? swapData.currency.value : currency.value}`} decimalScale={5} />
```
Comment (rocky-fleek) [inline source/ui/ActivityItem/index.jsx:186]: got an empty space in classname

Comment (ChriLnth) [reply source/ui/ActivityItem/index.jsx:186]: its intended to leave a space after the number

**Review by rocky-fleek — APPROVED** [review]

**Pull request merged**